### ISSUE TITLE
Update Material_File entries in DB and load.sql

### ISSUE THREAD

**Issue by grumpyinca** [thegrumpys/odop#271]
In multiple files (init, eqnset, etc.) change the current lines:
```javascript
     if (x[o.Material_File] === "mat_SI.json") m_tab = require('../mat_SI.json');
        else m_tab = require('../mat_ips.json'); 
```
to:

    m_tab = require('../' + x[o.Material_File]);

**Comment by grumpyinca** [comment]
Note that making this change will require changes to the Material_File entry in older database entries and corresponding load.sql entries.  Specifically, Spring/Compression Demo and Hot_Wnd need updating.

**Comment by grumpyinca** [comment]
Spring/Compression Demo and Hot_Wnd entries in both the development and production databases have now been updated.  load.sql still needs to be brought up to date.

**Comment by grumpyinca** [comment]
Spring/Compression load.sql was updated manually.  Merged to master; pushed to remote.  Released as part of Release 3.1.1.

The title of this issue is being changed to reflect the work done.  A new issue will be opened with the original title to cover the original intent.
